src/utils/addLine.js

```diff
@@ -1,29 +0,0 @@
-import * as THREE from "three";
-
-/**
- * addLine
- *
- * @type {Function} Draws line on the scene.
- *
- * @param {Object} scene
- * @param {Array} coor1
- * @param {Array} coor2
- * @param {Number} lineColor
- *
- */
-
-const addLine = (scene, coor1, coor2, lineColor) => {
```

Comment:
We might need a similar function in the future to show bonds between atoms but I think it's fine to remove this now.